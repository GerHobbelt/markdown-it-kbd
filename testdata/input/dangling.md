# Test

Here are some combinations to confuse the parser:
[[foo]] [[
[[bar]] ]] hey
[[this]] [[ [[and that]]
[[that]] ]] [[and this]]
[[ *some markup* [[ `more markup` [[valid]] **even more**
[[
[[*test*
