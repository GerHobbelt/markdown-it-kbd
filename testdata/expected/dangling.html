<h1>Test</h1>

<p>
	Here are some combinations to confuse the parser:
	<kbd>foo</kbd> [[
	<kbd>bar</kbd> ]] hey
	<kbd>this</kbd> [[ <kbd>and that</kbd>
	<kbd>that</kbd> ]] <kbd>and this</kbd>
	[[ <em>some markup</em> [[ <code>more markup</code> <kbd>valid</kbd> <strong>even more</strong>
	[[
	[[<em>test</em>
</p>
